spec/services/rights_service_spec.rb

```diff
@@ -1,17 +1,29 @@
 require 'spec_helper'
 
 describe RightsService do
-  describe "select_options" do
-    subject { described_class.select_options }
+  before do
+    # Configure QA to use fixtures
+    qa_fixtures = { local_path: File.expand_path('../../fixtures/authorities', __FILE__) }
+    stub_const("Qa::Authorities::LocalSubauthority::AUTHORITIES_CONFIG", qa_fixtures)
```

Comment:
We'll it's not optimal that QA doesn't have a better way to do this, but not your problem. 👍